rust/ql/test/library-tests/dataflow/sources/web_frameworks.rs

```diff
@@ -0,0 +1,199 @@
+#![allow(deprecated)]
+
+fn sink<T>(_: T) { }
+
+// --- tests ---
+
+mod poem_test {
+    use poem::{get, handler, web::Path, web::Query, Route, Server, listener::TcpListener};
+    use serde::Deserialize;
+    use crate::web_frameworks::sink;
```

Comment:
```suggestion
    use super::sink;
```
This has the same meaning but is a bit easier to read and write IMO.